pages/product.html

```diff
@@ -265,80 +254,72 @@ <h3 class="text-heading-2md font-roboto font-extrabold leading-tight">
-            <li class="cursor-pointer">Terms and Condition</li>
-            <li class="cursor-pointer">Blog</li>
-            <li class="cursor-pointer">Contact Us</li>
+          <ul class="footer__container-list">
+            <li class="footer__container-list-title">Resources</li>
+            <li>Privacy Policy</li>
+            <li>Terms and Condition</li>
+            <li>Blog</li>
+            <li>Contact Us</li>
           </ul>
           <ul class="flex flex-col gap-3">
             <li class="text-white font-bold paragraph-large">Product</li>
-            <li class="cursor-pointer">Project managment</li>
-            <li class="cursor-pointer">Time tracker</li>
-            <li class="cursor-pointer">Time schedule</li>
-            <li class="cursor-pointer">Lead generate</li>
-            <li class="cursor-pointer">Remote Collaboration</li>
+            <li>Project managment</li>
+            <li>Time tracker</li>
+            <li>Time schedule</li>
+            <li>Lead generate</li>
+            <li>Remote Collaboration</li>
           </ul>
 
-          <div class="flex flex-col gap-4 col-span-2">
-            <h1 class="font-roboto-bold text-white text-3xl">{finsweet</h1>
+          <div class="footer__subscribe">
+            <h1 class="footer__logo">{finsweet</h1>
 
-            <div class="w-full flex flex-col gap-4">
-              <p class="text-white font-bold paragraph-large">
-                Subscribe to our Newsletter
-              </p>
-              <div
-                class="flex gap-3 justify-between rounded-xsm about-subscribe w-full h-fit"
-              >
+            <div class="footer__subscribe-content">
+              <p class="footer__subscribe-title">Subscribe to our Newsletter</p>
+              <div class="footer__subscribe-input">
                 <input
                   type="text"
                   placeholder="Enter Your Email"
-                  class="outline-none px-4 font-roboto bg-transparent text-white"
+                  class="footer__subscribe-input-holder"
                 />
-                <button
-                  class="primary-btn primary-btn--fill-white h-full w-full"
-                >
+                <button class="primary-btn primary-btn--fill-white">
                   Get Free trial
                 </button>
               </div>
             </div>
           </div>
         </div>
-        <div class="flex justify-center mt-5">
-          <img src="../assets/vectors/Line.svg" class="w-full" />
-          <div class="w-full flex justify-around items-center">
-            <p class="text-xs">© Copyright Finsweet 2022</p>
-            <div class="flex gap-8">
+        <div class="footer__link">
+          <img src="/assets/vectors/Line.svg" class="footer__line" />
+          <div class="footer__link-content">
+            <p class="footer__link-text">© Copyright Finsweet 2022</p>
+            <div class="footer__link-space">
```

Comment:
here `footer__icons-container`, or `footer__social-icons`, etc... , improve this too